main.go

```diff
@@ -60,6 +60,7 @@ type options struct {
 	Patches        []string  `short:"p" long:"patch" value-name:"file"`
 	Diff           bool      `short:"d" long:"diff"`
 	DisplayVersion bool      `long:"version"`
+	Format         bool      `long:"stdout"`
```

Comment:
This field shouldn't be named Format.
It should be named "Stdout".

Although now that I think about it, both, the flag should be `--print` or `--print-only`?
```suggestion
	Print          bool      `long:"print-only"`
```